### PR TITLE
feat(buttons): created a renderdiv to render a person in a div

### PR DESCRIPTION
**Pull request by SharonStrats**
**Overview**
This PR adds a new function renderAsDiv which will render a person (image, name, and link) in a div.
1) includes renderAsDiv function
2) a bit of a refactor.
3) Tests
4) And a storybook example

No breaking changes.

### CODE REVIEW

File: src/stories/Interactive.stories.mdx [hunk 1]
```diff
@@ -55,10 +61,31 @@ A TR to represent a draggable person, etc in a list
...
       return UI.widgets.personTR(document, null, michiel)
-    }
-    }
+    }}
+  </Story>
+</Canvas>
+
+## renderAsDIV
+
+A Div to represent a draggable person, etc in a list
```
Comment (TallTed) [inline src/stories/Interactive.stories.mdx:74]:
```suggestion
A Div to represent (a) draggable person(s), etc., in a list
```

File: test/unit/widgets/buttons/iconLink.test.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  })
+  it('runs', () => {
+    const subject = new NamedNode('https://test.test#')
+    const iconURI = ''
+    expect(linkIcon(dom, subject, iconURI)).toBeTruthy()
+  })
+})
+
+describe('createLinkForURI', () => {
+  it('creates a Link for a URI element should return with two children a tag and br tag', () => {
```
Comment (TallTed) [inline test/unit/widgets/buttons/iconLink.test.ts:34]:
```suggestion
  it('creates a Link for a URI element; should return with two children: a tag and br tag', () => {
```

**Review by TallTed — CHANGES_REQUESTED** [review]

File: src/stories/Interactive.stories.mdx [hunk 3]
```diff
@@ -55,10 +61,31 @@ A TR to represent a draggable person, etc in a list
...
+
+[API documentation](https://solid.github.io/solid-ui/Documentation/api/modules/_widgets_buttons_.html#persontr)
+
+<Canvas>
```
Comment (angelo-v) [inline src/stories/Interactive.stories.mdx:78]: **praise** Thanks for adding a story.

**issue** It does not look well aligend, is this only in storybook or a general problem with the component?

File: src/stories/Interactive.stories.mdx [hunk 4]
```diff
@@ -55,10 +61,31 @@ A TR to represent a draggable person, etc in a list
...
-    }
+    }}
+  </Story>
+</Canvas>
+
+## renderAsDIV
+
+A Div to represent a draggable person, etc in a list
+
+[API documentation](https://solid.github.io/solid-ui/Documentation/api/modules/_widgets_buttons_.html#persontr)
```
Comment (angelo-v) [inline src/stories/Interactive.stories.mdx:76]: **issue** The documentation link is wrong, it should not point to persontr. Besides that all those links seem to be broken, perhaps you can update them

File: test/unit/widgets/buttons.test.ts [hunk 5]
```diff
@@ -186,7 +196,58 @@ describe('continueButton', () => {
...
+    createLinkDiv(dom, element, obj, options)
+    expect(element.getAttribute('draggable')).toEqual('true')
+  })
+  it('adds the style....', () => {
+    const options = {}
+    createLinkDiv(dom, element, obj, options)
+    expect(element.children[0].getAttribute('style')).toEqual(style.linkDivStyle)
+  })
+
+  it.skip('adds the deleteFunction of .... deleteButton with Check', () => {
```
Comment (angelo-v) [inline test/unit/widgets/buttons.test.ts:220]: **issue** this test should not be skipped
You are probably missing `deleteButton.setAttribute('role', 'button')` to make this test work

**Review by angelo-v — CHANGES_REQUESTED** [review]

**Comment by bourgeoa** [comment]
Sorry @angelo-v I did not see all comments and requested changes.
Having never tested reverting a PR I prefer to keep it.

@SharonStrats you can add your changes after merging

**Pull request merged**